pallets/capacity/src/tests/mock.rs

```diff
@@ -140,8 +143,8 @@ impl StakingRewardsProvider<Test> for TestStakingRewardsProvider {
 	type RewardEra = TestRewardEra;
 	type Hash = Hash; // use what's in common_primitives::node
 
-	fn reward_pool_size(total_staked: BalanceOf<Test>) -> BalanceOf<Test> {
-		total_staked.div(10u64)
+	fn reward_pool_size(_total_staked: BalanceOf<Test>) -> BalanceOf<Test> {
```

Comment:
To reflect new economic model behavior of having a constant RewardPool amount.

Reply:
Might also worth adding that as a comment in the code for the reference